### PR TITLE
Hannah kirrane branch

### PR DESCRIPTION
**Pull request by AMBA-Hannah-Kirrane**
## Proposed changes

My changes made in the training session

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have checked that my changes have not broken any other functionality
- [x] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?

@frankie-t-h

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 This is a random text file. You can add and remove things to it to test how git works.
 
+Here is my text file which I am editing on the training session this morning.
 
-This is some text.
-I am just normally editing a text file here.
+Hi Hannah, Jack here. Cheers to collaboration!
```
Comment (DavidThompson2) [inline text_file.txt:5]: Great to see you collaborating

**Review by DavidThompson2 — APPROVED** [review]
Looks good to me

File: text_file.txt (hunk 1 above)
Comment (LAURMURPH) [reply text_file.txt:5]: thumbs up

**Comment by LAURMURPH** [comment]
all good

**Pull request closed without merging**